pkg/manager/impl/execution_manager.go

```diff
@@ -672,6 +673,20 @@ func resolvePermissions(request *admin.ExecutionCreateRequest, launchPlan *admin
 	return &admin.AuthRole{}
 }
 
+func resolvePermissionsFromSecurityCtx(ctx context.Context, request admin.ExecutionCreateRequest, launchPlan *admin.LaunchPlan) *core.SecurityContext {
+	// Use security context from the request if its set
+	if request.Spec.SecurityContext != nil {
+		return request.Spec.SecurityContext
+	}
+
+	// Use launchplans security context if its set
+	if launchPlan.Spec.SecurityContext != nil {
+		return launchPlan.Spec.SecurityContext
+	}
+	logger.Warn(ctx, "Returning an empty security context")
```

Comment:
I think we should fall back on building Security Context from AuthRole if they are not specified in the request/launchplan... for the transition to deprecate AuthRole, I think Admin should be the place to make this translation...

Reply:
Agreed. Done